app/handler/trades.js

```diff
@@ -377,6 +377,7 @@ exports.createOffer = function (details, callback) {
                             } else if (err.message.indexOf('can only be sent to friends') != -1) {
                                 return callback(err);
                             } else if (err.message.indexOf('maximum number of items allowed in your Team Fortress 2 inventory') > -1) {
```

Comment:
Notifying on this error means that it is already too late, since it couldn't send the offer. It should alert us when it reaches 80-90% capacity instead, so we can withdraw items in time and avoid losing trades.